src/Integration.Vsix.UnitTests/Commands/ProjectTestPropertySetCommandTests.cs

```diff
@@ -33,150 +33,57 @@ namespace SonarLint.VisualStudio.Integration.UnitTests.Commands
     [TestClass]
     public class ProjectTestPropertySetCommandTests
     {
-        #region Test boilerplate
-
-        private ConfigurableVsProjectSystemHelper projectSystem;
-
-        // TODO - cleanup. These unit tests are supposed to be testing
-        // ProjectExcludePropertyToggleCommand, so they don't need a real
-        // instance of ProjectPropertyManager. See #4770.
-        private ProjectPropertyManager propertyManager;
-        private Mock<IProjectToLanguageMapper> projectToLanguageMapper;
-
-        [TestInitialize]
-        public void TestInitialize()
-        {
-            projectSystem = new ConfigurableVsProjectSystemHelper(new ConfigurableServiceProvider());
-            propertyManager = new ProjectPropertyManager(projectSystem);
-            projectToLanguageMapper = new Mock<IProjectToLanguageMapper>();
-        }
-
-        #endregion Test boilerplate
-
         #region Tests
 
         [TestMethod]
-        public void ProjectTestPropertySetCommand_Ctor()
+        public void Ctor_ArgsCheck()
         {
-            Action act = () => new ProjectTestPropertySetCommand(null, projectToLanguageMapper.Object, true);
+            Action act = () => new ProjectTestPropertySetCommand(null, Mock.Of<IProjectToLanguageMapper>(), true);
             act.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("propertyManager");
 
-            act = () => new ProjectTestPropertySetCommand(propertyManager, null, true);
+            act = () => new ProjectTestPropertySetCommand(Mock.Of<IProjectPropertyManager>(), null, true);
             act.Should().Throw<ArgumentNullException>().And.ParamName.Should().Be("projectToLanguageMapper");
         }
 
         [TestMethod]
-        public void ProjectTestPropertySetCommand_Invoke_SingleProject_SetsValue()
+        [DataRow(true)]
+        [DataRow(false)]
+        [DataRow(null)]
+        public void Invoke_MultipleProjects_SetsValues(bool? setPropertyValue)
```

Comment:
There was no real reason to test single project and multiple projects for this specific method so I removed the SingleProject one.